sdk/identity/azure-identity/test/ut/chained_token_credential_test.cpp

```diff
@@ -42,6 +42,8 @@ class TestCredential : public TokenCredential {
     token.Token = m_token;
     return token;
   }
+
+  std::string GetCredentialName() const override { return "TestCredential"; }
```

Comment:
Do you have a test which captures the case where this is not overridden, given this virtual is added after v1?

Reply:
Thanks, I added it.